gammapy/image/models/core.py

```diff
@@ -170,6 +172,136 @@ def evaluate(lon, lat, lon_0, lat_0, r_0):
         return u.Quantity(norm.value * (sep <= r_0), "sr-1", copy=False)
 
 
+class SkyEllipse(SkySpatialModel):
+    r"""Constant elliptical model
+
+    .. math::
+       \phi(\text{lon}, \text{lat}) =
+                \begin{cases}
+                    N & \text{for }  \,\,\,dist(F_1,P)+dist(F_2,P)\leq 2 a \\
+                    0 & \text{otherwise }\,,
+                \end{cases}
+
+
+
+    where :math:`F_1` and :math:`F_2` represent the foci of the ellipse (located by the method `find_foci()`),
+    :math:`P` is a generic point of coordinates :math:`(\text{lon}, \text{lat})`,
+    :math:`a` is the major semiaxis of the ellipse and N is the model's
+    normalization, in units of :math:`\text{sr}^{-1}`.
+    
+    The model is defined on the celestial sphere, by computing angles and distances with the functions defined in
+    `astropy.coordinates.angle_utilities`, and setting the normalization such that:
+
+    .. math::
+       \int_{4\pi}\phi(\text{lon}, \text{lat}) \,d\Omega = 1\,.
+
+
+    Parameters
+    ----------
+    lon_0 : `~astropy.coordinates.Longitude`
+        :math:`\text{lon}_0`: `lon` coordinate for the center of the ellipse.
+    lat_0 : `~astropy.coordinates.Latitude`
+        :math:`\text{lat}_0`: `lat` coordinate for the center of the ellipse.
+    semi_major : `~astropy.coordinates.Angle`
```

Comment:
Maybe it's an option to rename it to `r_0` to be consistent with `SkyDisk` model, for `e = 0`...I have no strong opinion here.

Reply:
Thank you for all your comments! So yes, using `r_0` might be an option. I think it might lead to some confusion, however. Indeed, if the model is wrongly passed the minor semiaxis it misbehaves